src/server/engine_shard.cc

```diff
@@ -757,62 +757,56 @@ void EngineShard::Heartbeat() {
 }
 
 void EngineShard::RetireExpiredAndEvict() {
-  {
-    FiberAtomicGuard guard;
```

Comment:
The bug diff here is just because I removed the the scooping for the FiberAtomicGuard
This check is now inside JournalFlushGuard